cirq/ops/fsim_gate.py

```diff
@@ -56,6 +56,12 @@ class FSimGate(gate_features.TwoQubitGate,
     """
 
     def __init__(self, theta: float, phi: float):
+        """
+        Args:
+            theta: The strength of the XX+YY interaction, in radians.
```

Comment:
(XX+YY) / 2 ?

Reply:
Maybe we should list which angles correspond to which operations, e.g. the angle at which 01 becomes 10 (phased) for the first time For now I'll merge this; feel free to improve it further.